Softeq.XToolkit.Bindings.iOS/Gestures/TapGestureRecognizerBehaviour.cs

```diff
@@ -6,7 +6,7 @@
 namespace Softeq.XToolkit.Bindings.iOS.Gestures
 {
     public class TapGestureRecognizerBehaviour
-        : GestureRecognizerBehavior<UITapGestureRecognizer>
+        : GestureRecognizerBehaviour<UITapGestureRecognizer>
```

Comment:
That's nice that you have found this inconsistency. But be careful with renaming classes in the library: some other code can relay on the old name. Probably we should create a class with the old name, derive it from this class and mark it Obsolete with some warning. @wcoder can provide more details, or maybe he knows we can just rename this class

Reply:
Renaming in this case doesn't really matter. We don't use this class directly.